cvat/apps/dataset_manager/util.py

```diff
@@ -80,3 +92,138 @@ def deepcopy_simple(v):
         return v
     else:
         return deepcopy(v)
+
+
+class LockNotAvailableError(Exception):
+    pass
+
+
+def make_export_lock_key(filename: os.PathLike[str]) -> str:
+    return f"export_lock:{filename}"
+
+@contextmanager
+def get_dataset_cache_lock(
```

Comment:
```suggestion
def get_export_lock(
```

This would be consistent with `make_export_lock_key`.

Reply:
Updated